solidity/scripts/lcl-client-config.js

```diff
@@ -44,9 +44,10 @@ module.exports = async function () {
       const formattedConfigFile = tomlify.toToml(fileContent, {
         space: 2,
         replace: (key, value) => {
-          return key == "Port" ? value.toFixed(0) : false
+          // Find keys that match exactly `Port` or ends with `MetricsTick`.
+          return key.match(/(^Port|MetricsTick)$/) ? value.toFixed(0) : false;
```

Comment:
When running the local install I am getting:
```
Updating keep-core client configs...
Using network 'local'.

failed to update keep-core client config TypeError: key.match is not a function
    at Object.replace (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/scripts/lcl-client-config.js:47:32)
    at escapeValue_ (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:223:39)
    at escapeArray (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:172:23)
    at escapeValue_ (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:241:16)
    at escapeKeyValue (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:276:18)
    at callback (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:493:20)
    at traverse (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:316:9)
    at traverse (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:350:13)
    at traverse (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:412:9)
    at escapeTable (/Users/piotr/go/src/github.com/keep-network/keep-core/solidity/node_modules/tomlify-j0.4/tomlify.js:511:5)
```

Reply:
Would you mind posting your config file content?

Reply:
Sure, nothing particularly scary there I think:
```
[ethereum]
  URL = "ws://127.0.0.1:8546"
  URLRPC = "http://127.0.0.1:8545"
  MiningCheckInterval = 15
  MaxGasPrice = 40000000000

[ethereum.account]
  KeyFile = "/Users/piotr/ethereum/data/keystore/UTC--2018-11-01T06-24-04.418691008Z--65ea55c1f10491038425725dc00dffeab2a1e28a"

[ethereum.ContractAddresses]
  BondedECDSAKeepFactory = "0xc0f93b172d863fEA888665BAfaDC4852A942816f"

[SanctionedApplications]
  Addresses = [
    "0x4f7C771Ab173bEc2BbE980497111866383a21172"
  ]

[Storage]
  DataDir = "/Users/piotr/keep-ecdsa-data/1"

[LibP2P]
  Port = 4921
```

Reply:
Fixed in 3225538